gtsam/basis/Chebyshev2.h

```diff
@@ -83,6 +86,11 @@ class GTSAM_EXPORT Chebyshev2 : public Basis<Chebyshev2> {
     return points;
   }
 
+  /// Return a zero initialized Parameter matrix.
+  static Parameters ParameterMatrix(size_t N) {
```

Comment:
I don't see why we can't just say Zero(N) ?

Reply:
Yes this needs to be removed since it was added for the previous change of `N` to `N+1`. Removing.